.github/workflows/create-release.yml

```diff
@@ -58,12 +59,19 @@ jobs:
           git config --global user.email '${{ steps.get-user-id.outputs.user-id }}+${{ steps.generate_token.outputs.app-slug }}@users.noreply.github.com'
           ./.tools/multimod tag --module-set-name liatrio-otel --commit-hash '${{ github.sha }}' --print-tags
           git push --tags
-
+      - name: Get changelog
+        if: steps.get_next_version.outputs.hasNextVersion == 'true'
+        id: get_tag
+        uses: mathieudutour/github-tag-action@a22cf08638b34d5badda920f9daf6e72c477b07b # v6.2
+        with:
+          github_token: ${{ steps.generate_token.outputs.token }}
```

Comment:
Do we want this token? I don't think it will right now but given this is a GH app token it will trigger other Action events.

Reply:
Switched to use the default GITHUB_TOKEN for security and simplicity